R/build-news.R

```diff
@@ -321,6 +324,17 @@ tweak_news_heading <- function(x, versions, timeline, bs_version) {
   invisible()
 }
 
+add_version <- function(header, version) {
+  version <- gsub("\\s+", "-", version)
```

Comment:
Looks like it's getting an extra `#`, e.g. https://608c17c7e8d7752b4dda2b48--pkgdown-dev.netlify.app/news/index.html##major-changes-1.6.0

Reply:
Ouch, now fixed.